libretroshare/src/retroshare/rsconfig.h

```diff
@@ -180,11 +202,22 @@ class RSTrafficClue
 
     RSTrafficClue() { TS=0;size=0;service_id=0;service_sub_id=0; count=0; }
     RSTrafficClue& operator+=(const RSTrafficClue& tc) { size += tc.size; count += tc.count ; return *this ;}
+
+	// RsSerializable interface
+public:
```

Comment:
public can be removed here as it is a struct and averything is public by default